packages/components/spinner/src/spinner.js

```diff
@@ -0,0 +1,47 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import '@tradeshift/elements.cover';
+import css from './spinner.css';
+import { types, sizes, colors } from './utils';
+
+customElementDefineHelper(
+	'ts-spinner',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				color: { type: String, attribute: 'data-color' },
```

Comment:
We need to investigate more and decide about this one in the future as well if we want to have this prefix on every properties or not

Reply:
the color name is also can be misleading and they can just think they can pass any color, what do you think?
Maybe we can just remove the light/dark and just use the type for defining the color.